core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXV1Endpoint.java

```diff
@@ -197,6 +198,12 @@ void deleteByCrn(@ValidCrn(resource = CrnResourceDescriptor.DATAHUB) @PathParam(
             nickname = "retryDistroXV1ByCrn")
     void retryByCrn(@ValidCrn(resource = CrnResourceDescriptor.DATAHUB) @PathParam("crn") String crn);
 
+    @POST
+    @Path("crn/{crn}/cancel/internal")
```

Comment:
cancelling what? we need a more precise name for this endpoint and method too

Reply:
Also meant to be an API for use during development. Should not go in, not without limiting access to it anyway.
The intent here was: During development, if a flow gets stuck, how do you reset the state of the cluster to continue testing?
Is there a better approach to this problem?
Otherwise maybe we could think of this temporary API in the future (cancels flows on a cluster, and resets state) behind some kind of a dev entitlement.
For now, will remove.

Reply:
unfortunately I am not aware of any easy way to cancel flow, I usually manipulating my database during database to make a flow finished/failed

Reply:
Fixed in the next push by removing this API for now. Will file a separate jira to see if there's a good way to introduce something similar for easier development. Resolving conversation.